### DISCUSSION TITLE
Welcome to reactjs-popup Discussions!

### DISCUSSION THREAD

**Discussion by yjose** [yjose/reactjs-popup#356] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by HussLinHq** [comment]
Hi Guys, 
We have had series of discussions via email about the need to do some fixes and upgrades, well the opportunities present itself to do so now.
I believe the best way to start is to do some fixes first on some positions we talked about via email and then move onto upgrades.

**Reply by HussLinHq** [reply]
Some of the proposed fixes and upgrades would include: 
*Support for Controlled Components,
*Improved Responsiveness and Scalability,
*Customizable Animations,
*Enhanced Documentation and Examples
